File: TP3_Python/Codingan/MOUSE.py
from External import External

class MOUSE(External):
    def __init__(self, tipe_mouse="", kecepatan=0, koneksi="", merk_external=""):
        super().__init__(koneksi, merk_external)
        self.tipe_mouse = tipe_mouse
        self.kecepatan = kecepatan

    def set_tipe_mouse(self, tipe_mouse):
        self.tipe_mouse = tipe_mouse

    def set_kecepatan(self, kecepatan):
        self.kecepatan = kecepatan

    def get_tipe_mouse(self):
        return self.tipe_mouse

    def get_kecepatan(self):
        return self.kecepatan

File: TP3_CPP/Codingan/MOUSE.cpp
#pragma once
#include "External.cpp"
#include <iostream>
#include <string>

using namespace std;

class MOUSE : public External
{
private:
    string TipeMouse;
    int Kecepatan;

public:
    MOUSE()
    {
    }

    MOUSE(string TipeMouse, int Kecepatan, string Koneksi, string MerkExternal) : External(Koneksi, MerkExternal)
    {
        this->TipeMouse = TipeMouse;
        this->Kecepatan = Kecepatan;
    }

    // Setter
    void setTipeMouse(string TipeMouse)
    {
        this->TipeMouse = TipeMouse;
    }

    void setKecepatan(int Kecepatan)
    {
        this->Kecepatan = Kecepatan;
    }

    // Getter
    string getTipeMouse()
    {
        return TipeMouse;
    }

    int getKecepatan()
    {
        return Kecepatan;
    }

    ~MOUSE()
    {
    }
};
